### PR TITLE
[CLEANUP] Code cleanup

### PR DESCRIPTION
**Pull request by CapslockRSPS**
Just some code cleanup.

### CODE REVIEW

File: src/main/java/org/rev317/min/api/methods/Game.java [hunk 1]
```diff
@@ -146,9 +146,10 @@ public static boolean confirmedDropClient() {
         Time.sleep(new SleepCondition() {
             @Override
             public boolean isValid() {
-                return isLoggedIn();
+                return !isLoggedIn();
```
Comment (JKetelaar) [inline src/main/java/org/rev317/min/api/methods/Game.java]: Why?

File: src/main/java/org/rev317/min/accessors/Interface.java [hunk 2]
```diff
@@ -6,13 +6,7 @@
 
     int[] getStackSizes();
 
-    /**
-     * This is meant for the clients that have a long value as their stacksizes, simply because they have no logic...
-     *
-     * @return long version of #getStackSizes
```
Comment (JKetelaar) [inline src/main/java/org/rev317/min/accessors/Interface.java]: Keep this return tag

Comment (JKetelaar) [inline src/main/java/org/rev317/min/accessors/Interface.java]: Rest can be removed though

File: src/main/java/org/rev317/min/api/methods/Game.java (hunk 1 above)
Comment (CapslockRSPS) [inline src/main/java/org/rev317/min/api/methods/Game.java]: Look at the code and tell me if it made sense.

File: src/main/java/org/rev317/min/accessors/Interface.java (hunk 2 above)
Comment (CapslockRSPS) [inline src/main/java/org/rev317/min/accessors/Interface.java]: Okay.

**Pull request merged**